README.rst

```diff
@@ -43,33 +47,79 @@ The primary use cases are:
 `The documentation <https://jupyter-server-proxy.readthedocs.io/>`_
 contains information on installation & usage.
 
-====================
 Install
-====================
+=======
+
+Python package
+--------------
 
 pip
----
+^^^
 
 .. code-block::
 
    pip install jupyter-server-proxy
 
 conda
------
+^^^^^
 
 .. code-block::
 
    conda install jupyter-server-proxy -c conda-forge
 
-src
----
+JupyterLab extension
+--------------------
+
+Note that as the JupyterLab extension only is a graphical interface to
+launch registered applications in the python package, the extension
+requires the python package to be installed.
 
 .. code-block::
 
-   pip install .
+   jupyter labextension install @jupyterlab/server-proxy
+
+Contributing
+============
+
+Python package
+--------------
+
+.. code-block::
+
+   pip install -e .
+
+   # explicit install needed with editable mode (``-e``) jupyter
```

Comment:
```suggestion
   # explicit install needed with editable mode (-e) jupyter
```
This is already inside a code-block:
![image](https://user-images.githubusercontent.com/1644105/78180964-76ab3880-745b-11ea-96c8-a79f04f4ab0c.png)